operator/src/main/java/oracle/kubernetes/operator/helpers/ConfigMapHelper.java

```diff
@@ -504,7 +504,11 @@ public NextAction apply(Packet packet) {
     }
 
     private long timeSinceJobStart(Packet packet) {
-      return System.currentTimeMillis() - ((Long) packet.get(JobHelper.START_TIME));
+      return System.currentTimeMillis() - getStartTime(packet);
+    }
+
+    private Long getStartTime(Packet packet) {
+      return Optional.ofNullable((Long) packet.get(JobHelper.START_TIME)).orElse(0L);
```

Comment:
It probably ok, but if START_TIME is not set, then  the elapsed time will be very long.

Reply:
This check is for the operator restart use case where the introspector job is started before the operator is restarted. The START_TIME will not be set in the packet in this case. It seems the elapsed time is only used to log a FINE level log message and I'm not sure if it's worth storing the START_TIME in the introspector config map if it's only used for logging.

Reply:
I have removed this check by setting the START_TIME to job creation time for operator restart use-case.